src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_NullableHelpers.vb

```diff
@@ -182,13 +178,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         If IsConditionalAccess(expr, whenNotNull, whenNull) AndAlso HasNoValue(whenNull) Then
                             Debug.Assert(Not HasNoValue(whenNotNull))
                             Return UpdateConditionalAccess(expr,
-                                                           NullableValueOrDefault(whenNotNull),
+                                                           NullableValueOrDefault(whenNotNull, isOptional),
```

Comment:
>NullableValueOrDefault(whenNotNull, isOptional) [](http://example.com/codeflow?start=59&length=47)

Simply inlining the value doesn't look appropriate, We don't want to create conditional access with null value, we want to return null value instead.